.gitignore

```diff
@@ -13,3 +13,5 @@ autoCD.py
 CS851Local.py
 cacheService.py
 surt/
+venv/
+.cache
```

Comment:
Add a newline at the end of the file (it is not required for functional reasons, but it is a god practice in general to avoid issues when files are concatenated).